lib/googleauth/signet.rb

```diff
@@ -109,14 +110,27 @@ def build_default_connection
         end
       end
 
+      # Retries the provided block with exponential backoff, handling and wrapping errors.
+      #
+      # @param [Integer] max_retry_count The maximum number of retries before giving up
+      # @yield The block to execute and potentially retry
+      # @return [Object] The result of the block if successful
+      # @raise [Google::Auth::AuthorizationError] If a Signet::AuthorizationError occurs or if retries are exhausted
+      # @raise [Google::Auth::ParseError] If a Signet::ParseError occurs during token parsing
+      # rubocop:disable Metrics/MethodLength
```

Comment:
To prevent the rubocop directive from appearing in the documentation, move this line above the method documentation comments, separated by a blank line.

Reply:
fixed